pkgs/by-name/dm/dmt-core/package.nix

```diff
@@ -0,0 +1,71 @@
+{ lib, buildPythonApplication, fetchurl, fetchPypi, pythonPackages }:
+
+let
+  reprint = buildPythonApplication rec {
```

Comment:
is this a python module? and in any case, why not package it separately?

Reply:
It is. Which is why my initial commit was an overlay. It's used as a dependency. I don't see it as necessary for the time being.

Reply:
Never said anything about an overlay, overlays are for when you can't just modify nixpkgs itself - but we are in the nixpkgs repo, we can modify nixpkgs itself.

I'm saying these are probably best packaged in separate files.

Reply:
I'm receiving conflicting feedback in that case because my initial PR didn't use pkgs/by-name. The description even notes that it's a lib. I'm not sure if anyone is reading it. I'll make those modifications.

Reply:
There's no conflicting feedback here. Nobbz said

> prefer pkgs/by-name
> unless this is really a python package, then use pythonPackages

The phrasing to use `pythonPackages` was incorrect, it should've been `python3Packages`. Other than that, we are in agreement; if it's used for its modules, as I said earlier, then it stays in python-modules. If it's simply a binary, then it goes in pkgs/by-name.

So you need dmt-core in pkgs/by-name using buildPythonApplication, while reprint and verilogae go in python-modules using buildPythonPackage, if I understood the purpose of these packages correctly.

Reply:
> The phrasing to use pythonPackages was incorrect, it should've been python3Packages

Yeah, my fault, `pythonPackages` belong in `python2-modules` and `python3Packages` in `python-modules`, I hate whoever came up with that…